tasks/main.yml

```
@@ -1,2 +1,16 @@
 ---
-# tasks file for skeleton
+- name: Create the /var/cyhy/admiral directory
+  ansible.builtin.file:
+    group: "{{ admiral_file_owner_group | default(omit) }}"
+    mode: "0o755"
+    owner: "{{ admiral_file_owner_username | default(omit) }}"
+    path: /var/cyhy/admiral
+    state: directory
+
+- name: Install the Docker Compose configuration
+  ansible.builtin.copy:
+    dest: /var/cyhy/admiral
+    group: "{{ admiral_file_owner_group | default(omit) }}"
+    mode: "0o644"
```

We have been using a different syntax for `mode` for a while now: 
```suggestion
    mode: u=rw,g=r,o=r
```